internal/configs/iut/config.go

```diff
@@ -0,0 +1,146 @@
+// Copyright 2022 Axis Communications AB.
+//
+// For a full list of individual contributors, please see the commit history.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package config
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/sirupsen/logrus"
+)
+
+// Config interface for retrieving configuration options.
+type Config interface {
+	ServiceHost() string
+	ServicePort() string
+	LogLevel() string
+	LogFilePath() string
+	Timeout() time.Duration
+	ETOSNamespace() string
```

Comment:
Is this parameter used?

Reply:
ETOSNamespace() is needed to make IUT config conform to the base config interface in order to reuse the application interface. Timeout is removed.